src/server/debug/server/server.go

```diff
@@ -101,9 +102,74 @@ func NewDebugServer(env Env) debug.DebugServer {
 		logLevel:      log.LogLevel,
 		grpcLevel:     log.GRPCLevel,
 	}
+	ticker := time.NewTicker(3 * time.Second)
```

Comment:
I think that this should be configurable, or at the very least a package constant.